hooks/hostname.hook/README.md

```diff
@@ -1,49 +1,45 @@
 ## Sample counting hook
 
+You can use a hook to create more advanced dynamic hostnames than the simple incremented pattern --- `$\{id\}.example.com` --- from the `hostname` property on a policy. This type of hook calculates the correct hostname and returns that hostname as metadata on the task. To do so, it uses a basic counter system that stores how many nodes have bound to a given policy.
+
 ### Prerequisites
 
-This hook requires that ruby is installed. This must be contained in $PATH in
-order for the hook script to succeed.
+Ruby must be installed in `$PATH` for the hook script to succeed.
```

Comment:
Add:
"If it is not included, add it via the `hook_execution_path` property in the config.yaml file."